service/controller/resource/alert/create.go

```diff
@@ -0,0 +1,59 @@
+package alert
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/prometheus-meta-operator/service/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cluster, err := key.ToCluster(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	rules, err := getRules(obj, r.installation)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	r.logger.LogCtx(ctx, "level", "debug", "message", "ensuring the prometheus rules exists")
+	for _, desiredRule := range rules {
+		ruleName := desiredRule.GetName()
+		r.logger.LogCtx(ctx, "level", "debug", "message", "checking if rule needs to be created or updated", "rule", ruleName)
```

Comment:
We do not use separate keys to log additional information, but put everything under the `message` key. see https://github.com/giantswarm/fmt/blob/master/go/log_messages.md

Reply:
Alright, let me fix it before merging if circle ci allows me to